README.md

```diff
@@ -60,22 +60,36 @@ Provides a JupyterHub Service on an existing Openstack Cluster. This uses the he
 - Some metrics can't be selected by node name in Grafana dashboard as it requires a reverse DNS.
 
 ## Requirements
+The following assumes you have an Ubuntu 20.04 machine with `pip3`, `python3` and `python3-venv` already installed.
 
-- Ansible ([Installing Ansible — Ansible Documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html))
+
+- Ansible ([Installing Ansible — Ansible Documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)) - this is installed using `pip` in `py-requirements.txt`
 - Helm 3 ([Installing Helm](https://helm.sh/docs/intro/install/))
 - kubectl ([Install Tools | Kubernetes](https://kubernetes.io/docs/tasks/tools/))
-- Python 3
-  - `ansible, kubernetes`
+
+Helm and kubectl can be installed using snap:
+
+```bash
+sudo apt-get update && sudo apt-get install -y snapd
+export PATH=$PATH:/snap/bin
+sudo snap install kubectl --classic
+sudo snap install helm --classic
+```
+
 ### Local Environment Setup
-- Upgrade pip3 as the default version is too old to handle the required deps: `pip3 install pip --upgrade`
-- Activate .venv if present then install pip deps: `pip3 install ansible setuptools setuptools-rust pyyaml kubernetes`
-- Clone the repository and cd into it
-- Install requirements `ansible-galaxy collection install -r requirements.yml`
+
+- Upgrade `pip3` using `pip3 install pip --upgrade` to ensure you are using the latest version of pip.
+- Create a virtual environment `venv` using `python3 -m venv venv`
+- Activate `venv`
+- Install the python dependencies: `pip3 install -r py-requirements.txt`
+- If you get an error about the version of setuptools, upgrade it manually using `pip3 install setuptools --upgrade`
+- Clone this repository and `cd` into it
+- Install Ansible requirements `ansible-galaxy collection install -r requirements.yml`
 
 ## GPU
 
 - GPU drivers will automatically be pulled from the Nvidia NGC catalogue
-- Driver version is defined in playbooks/deploy_jhub.yml
+- Driver version is defined in `playbooks/deploy_jhub.yml`
```

Comment:
This can be completely removed, since the playbook doesn't control anything about drivers anymore
(Including the code this relates to in the future)

Reply:
Would the reference to the driver version need to be removed as well from `playbooks/deploy_jhub.yml`? Currently there are variable in the playbook that has a GPU driver versioon